### DISCUSSION TITLE
How to store multiple checkbox value inside table?

### DISCUSSION THREAD

**Discussion by vaibhavpanchal2218** [livewire/livewire#4482] [1. Help] [answered]
I have multiple checkboxes with the same name, I want to store their value into a database by using studentId. How Can I do it?

`public $termsMonth = [];
`
`input type="checkbox" wire:model="termsMonth" value="September" id="september" class="form-check-input"...
`
Validation working fine... But I want to store them into table.

**Comment by joshhanley** [comment, marked as answer]
@vaibhavpanchal2218 I'm not sure what the issue is exactly, if you should just use normal Laravel for storing data in the database, so you would take you property, and add its value to a model and save the model into the database.

Have you tried that? If so can you share some code so someone can have a look and see if they can help you out?

Thanks!

**Reply by vaibhavpanchal2218** [reply]
Hello Mr. Josh, Thank you for your reply. But I am done by using controller functions.

**Reply by joshhanley** [reply]
No worries! Ok great 🙂
